include/xsimd/types/xsimd_neon_uint8.hpp

```diff
@@ -349,6 +349,13 @@ namespace xsimd
                 return vcombine_u8(tmp.val[0], tmp.val[1]);
 #endif
             }
+
+            static batch_type extract_pair(const batch_type& lhs, const batch_type& rhs, const int n)
+            {
+                const int index = n;
```

Comment:
this assignment looks odd to me ^^!

Reply:
Yep, I added it to try to fix build issue as below which was not reproduced in my local: environment.: 
```
/home/vsts/work/1/s/include/xsimd/types/xsimd_neon_uint8.hpp:355:24: error: argument to '__builtin_neon_vextq_v' must be a constant integer
2021-04-27T09:53:57.3826864Z                 return vextq_u8(lhs, rhs, n);
```
But it also doesn't work.